__tests__/classValidatorToJsonSchema.test.ts

```diff
@@ -0,0 +1,53 @@
+import { IsString, MinLength } from 'class-validator'
+import { classValidatorToJsonSchema, validationMetadatasToSchemas } from '../src'
+
+export class User {
+  @MinLength(5)
+  @IsString()
+  name: string;
+}
```

Comment:
Is it possible to use namespaces in this file to import two classes with the same name? Might be clearer than declaring one here and another in a separate file.

I mean something like
```typescript
namespace Foo { export class User { } }
namespace Bar { export class User { } }
```